packages/api/src/beacon/routes/validator.ts

```diff
@@ -550,24 +550,31 @@ export function getReqSerializers(): ReqSerializers<Api, ReqTypes> {
   );
 
   const produceBlockV3: ReqSerializers<Api, ReqTypes>["produceBlockV3"] = {
-    writeReq: (slot, randaoReveal, graffiti, skipRandaoVerification, opts) => ({
-      params: {slot},
-      query: {
+    writeReq(slot, randaoReveal, graffiti, skipRandaoVerification, opts) {
+      const query: ReqTypes["produceBlockV3"]["query"] = {
         randao_reveal: toHexString(randaoReveal),
         graffiti: toGraffitiHex(graffiti),
         fee_recipient: opts?.feeRecipient,
-        skip_randao_verification: skipRandaoVerification,
         builder_selection: opts?.builderSelection,
         builder_boost_factor: opts?.builderBoostFactor?.toString(),
         strict_fee_recipient_check: opts?.strictFeeRecipientCheck,
         blinded_local: opts?.blindedLocal,
-      },
-    }),
+      };
+
+      if (skipRandaoVerification) {
+        query["skip_randao_verification"] = "";
+      }
```

Comment:
As per the [current spec](https://github.com/ethereum/beacon-APIs/blob/425824e42ff55a4e3c06ab2d434232d8ea2b543e/apis/validator/block.v3.yaml#L42-L43) 

> This query parameter is a flag and does not take a value.

So we have to change the type to `string` and make it only take `empty string` when the value is actually true.